GeoQuiz/app/src/main/java/com/example/geoquiz/QuizActivity.kt

```diff
@@ -0,0 +1,95 @@
+package com.example.geoquiz
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.Gravity
+import android.widget.Button
+import android.widget.ImageButton
+import android.widget.TextView
+import android.widget.Toast
+
+class QuizActivity : AppCompatActivity() {
+
+    private lateinit var mTrueButton: Button
```

Comment:
I assume these are the namings used in the Java edition of the book, right? :)

Reply:
Yes, they are from Java edition.